tests/cli_unittest.py

```diff
@@ -546,3 +546,112 @@ def test_ouputs_colored_dots_with_terminal(self):
                 + "\r\n"
             ),
         )
+
+
+class TestCliLongFormatter(TestCliBase):
+    def setUp(self):
+        super().setUp()
+        self.argv = ["--format", "long"] + self.argv
+
+    def test_colored_output_to_terminal(self):
+        """
+        Execute all examples in the order defined with colored output.
+        """
+        self.run_testslide(
+            tty_stdout=True,
+            expected_return_code=1,
+            expected_stdout_startswith=(
+                self.white("top context: ")
+                + self.green("passing example")
+                + "\r\n"
+                + self.white("top context: ")
+                + self.red("failing example: ")
+                + "SimulatedFailure: test failure (extra)"
+                + "\r\n"
+                + self.white("top context: ")
+                + self.green("*focused example")
+                + "\r\n"
+                + self.white("top context: ")
+                + self.yellow("skipped example")
+                + "\r\n"
+                + self.white("top context: ")
+                + self.yellow("unittest SkipTest")
+                + "\r\n"
+                + self.white("top context, nested context: ")
+                + self.green("passing nested example")
+                + "\r\n"
+                + self.white("tests.sample_tests.SampleTestCase: ")
+                + self.red("test_failing: ")
+                + "AssertionError: "
+                + "\r\n"
+                + self.white("tests.sample_tests.SampleTestCase: ")
+                + self.green("test_passing")
+                + "\r\n"
+                + self.white("tests.sample_tests.SampleTestCase: ")
+                + self.yellow("test_skipped")
+                + "\r\n"
+                # TODO Rest of the output
```

Comment:
Is this really a `TODO` or just `...rest of the output`?

Same for the other tests...

Reply:
It is a TODO. The output at this point contains timing information, which would require using regex, which's a pain with colored output. It is currently a compromise: covering 90% of features with 10% of effort.

Reply:
Thanks for the context. Maybe we could add this info to the comment to make it more clear?